python/GafferUI/CompoundEditor.py

```diff
@@ -290,6 +290,14 @@ def __captureEditorState( self ) :
 					"driver" : self.__pathToEditor(driver),
 					"driverMode" : mode
 				}
+			else :
+				nodeSet = n.getNodeSet()
+				# We don't want to couple the layout-centric serialisation that assumes
```

Comment:
Is this comment trying to explain why we're not using `repr( nodeSet )`?

Reply:
Improved in https://github.com/GafferHQ/gaffer/pull/3360/commits/45bc63ed9e4b00322775a27def17db7bb41f95c8